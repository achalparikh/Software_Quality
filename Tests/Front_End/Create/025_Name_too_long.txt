login
admin
create
Jeff Doe
100000.00
logout