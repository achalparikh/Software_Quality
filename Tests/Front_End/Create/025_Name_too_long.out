Login to start:
Enter type of account: "standard" or "admin"
You are now logged in as an admin
Enter a command:
Enter an account holder:
Enter a balance for the new account:
Error, account balance too large, could not create account
Enter a command:
You are now logged out
Login to start:
